src/ast.rs

```diff
@@ -2276,9 +2288,11 @@ impl<'a> GetLeafName<'a> for UnqualifiedName {
     fn get_leaf_name(&'a self, subs: &'a SubstitutionTable) -> Option<LeafName<'a>> {
         match *self {
             UnqualifiedName::ABITag(_)
-            | UnqualifiedName::UnnamedType(_)
-            | UnqualifiedName::Operator(_)
+            | UnqualifiedName::Operator(_) => None,
```

Comment:
Does this need `=> None,`? Shouldn't the `CtorDtor` arm below handle this?

Sort surprised that one can have a `|` before a match arm when there is no previous arm being merged into it...